mlos_bench/mlos_bench/tests/optimizers/manual_opt_test.py

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""Unit tests for mock mlos_bench optimizer."""
+
+from mlos_bench.environments.status import Status
+from mlos_bench.optimizers.manual_optimizer import ManualOptimizer
+
+# pylint: disable=redefined-outer-name
+
+
+def test_manual_optimizer(manual_opt: ManualOptimizer, mock_configs: list) -> None:
+    """Make sure that manual optimizer produces consistent suggestions."""
+
+    for i in range(2 * len(mock_configs)):
```

Comment:
Would be probably easier to maintain if you kept that dynamic (i.e., didn't expect coordination between the definition in the other file and this one).

Reply:
Updated this to rely on `while manual_opt.not_converged():`, so no more coordination in the lengths!